metadata-models/src/main/pegasus/com/linkedin/ml/metadata/MLModelGroupProperties.pdl

```diff
@@ -31,15 +31,4 @@ record MLModelGroupProperties includes CustomProperties {
    * Version of the MLModelGroup
    */
   version: optional VersionTag
-
-  /**
-   * Models in the group
-   */
-  @Relationship = {
-    "/*": {
-      "name": "Includes",
-      "entityTypes": [ "mlModel" ]
-    }
-  }
-  models: array[Urn] = [ ]
```

Comment:
Is this backward compatible?

Reply:
Note needed – model groups haven't been released yet!